elytron/src/test/java/org/wildfly/extension/elytron/RealmsTestCase.java

```diff
@@ -145,6 +145,50 @@ public void testOAuth2Realm() throws Exception {
         Assert.assertNotNull(securityRealm);
     }
 
+    @Test
+    public void testAggregateRealm() throws Exception {
+        KernelServices services = super.createKernelServicesBuilder(new TestEnvironment()).setSubsystemXmlResource("realms-test.xml").build();
+        if (!services.isSuccessfulBoot()) {
+            Assert.fail(services.getBootError().toString());
+        }
+
+        ServiceName serviceName = Capabilities.SECURITY_REALM_RUNTIME_CAPABILITY.getCapabilityServiceName("AggregateRealmOne");
+        SecurityRealm securityRealm = (SecurityRealm) services.getContainer().getService(serviceName).getValue();
+        Assert.assertNotNull(securityRealm);
+
+        RealmIdentity identity1 = securityRealm.getRealmIdentity(fromName("firstUser"));
+        Assert.assertTrue(identity1.exists());
+        //Assert that transformation was successful and the correct identity and attributes were loaded from filesystem-realm-2
```

Comment:
I think this comment can be removed here since this test doesn't use the transformer.

Reply:
removed